language/stdlib/src/lib.rs

```diff
@@ -196,7 +259,7 @@ pub fn build_packed_types_map() {
     path.push(PACKED_TYPES_FILENAME);
     path.set_extension(PACKED_TYPES_EXTENSION);
     options.output_path = path.to_str().unwrap().to_string();
-    let mut sources = stdlib_files();
+    let mut sources = stdlib_source_files();
```

Comment:
Shouldn't this be the source files for the LF and not the standard library?

Reply:
Good catch!!